instrumentation/otel-webserver-module/include/core/api/AppdynamicsSdk.h

```diff
@@ -27,6 +27,7 @@
 /* {{{ For API user: environment variables to specify Controller connectivity */
 #define OTEL_SDK_ENV_OTEL_EXPORTER_TYPE "OTEL_SDK_ENV_OTEL_EXPORTER_TYPE"
 #define OTEL_SDK_ENV_OTEL_EXPORTER_ENDPOINT "OTEL_SDK_ENV_OTEL_EXPORTER_ENDPOINT"             /*required*/
+#define OTEL_SDK_ENV_OTEL_EXPORTER_OTELHEADERS "OTEL_SDK_ENV_OTEL_EXPORTER_OTELHEADERS"             /*optional*/
```

Comment:
Can we have a better name here? to many OTEL
may be something below
OTEL_SDK_ENV_OTEL_EXPORTER_OTLPHEADERS